### PR TITLE
Update Wealth Distribution Dynamics Lecture

### LINKED ISSUE #169: Update wealth distribution lecture
- [ ] Replace `wealth_dynamics` with https://github.com/QuantEcon/imf_2024/blob/main/tuesday/morning/wealth_dynamics.ipynb
- [ ] Modify that lecture so that it uses myst -- see, for example, the exercises.

### PR DESCRIPTION
**Pull request by HumphreyYang**
This PR resolves #169.

### CODE REVIEW

**Comment by mmcky** [comment]
@HumphreyYang this will need a pip install of quantecon

```
No module named 'quantecon'
```

to fix the execution issue.

File: lectures/wealth_dynamics.md [hunk 1]
```diff
@@ -35,600 +49,752 @@ Let's check the GPU we are running
...
+#### Gini Coefficient
 
-Let's finish this section by investigating what happens when we change the
-volatility term $\sigma_r$ in financial returns.
+Recall that, for sorted data $w_1, \ldots, w_n$, the Gini coefficient takes the form
 
-```{code-cell} ipython3
-%%time
+\begin{equation}
+    \label{eq:gini}
```
Comment (mmcky) [inline lectures/wealth_dynamics.md:534]: @HumphreyYang I don't think that this label will get parsed. 

It would be better to wrap the latex in `$$` or `math` directive with a label `()` attribute

**Comment by mmcky** [comment]
thanks @HumphreyYang just some minor comments and fixes required.

**Review by mmcky — APPROVED** [review]

**Comment by jstac** [comment]
Thanks @HumphreyYang.

Is this based on an old version of main?  The TOC is not up to date

https://jax.quantecon.org/intro.html
https://660b3591fe556334aa88cf97--incomparable-parfait-2417f8.netlify.app/intro.html

**Comment by HumphreyYang** [comment]
Many thanks @jstac, 

Yes, this PR was opened before the TOC change was merged. I will merge the main into this branch and generate a new preview.

**Comment by jstac** [comment]
Thanks @HumphreyYang 
CC @mmcky

**Comment by mmcky** [comment]
thanks @HumphreyYang 

@jstac this looks good. Would you like this merged for Chile?

Wow! JAX is 290x faster. That's impressive!

**Comment by jstac** [comment]
Many thanks @HumphreyYang . Merging.
